src/main/groovy/com/github/prokod/gradle/crossbuild/tasks/AbstractCrossBuildPomTask.groovy

```diff
@@ -30,6 +30,7 @@ abstract class AbstractCrossBuildPomTask extends DefaultTask {
     static enum ScopeType {
         COMPILE,
         RUNTIME,
-        PROVIDED
+        PROVIDED,
+        COMPILE_ONLY
```

Comment:
Why did you add COMPILE_ONLY scope ? those are enumeration strictly for maven side